W-14143826 CG 5.0 release (#35)
mulesoft/docs-accelerators: consumer-goods/5.0/modules/ROOT/pages/release-notes.adoc

= Release Notes
:cg-version: 5.0

== 5.0

> *Sep 19, 2023*

Reviewer: ```suggestion
*September 19, 2023*
```

=== New use case

> This release includes the https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%204%20-%20Trade%20promotion%20optimization/[Trade Promotion Optimization^] use case, which helps integrate predictive baseline data from CRM Analytics into Hyperforce for trade promotion optimization, enabling data-driven insights into promotion effectiveness, aiding in designing targeted promotions aligned with consumer behavior, and facilitating accurate ROI calculations by assessing market risks.

Reviewer: ```suggestion
This release introduces the https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%204%20-%20Trade%20promotion%20optimization/[Trade Promotion Optimization^] use case, which integrates predictive baseline data from CRM Analytics into Hyperforce to optimize trade promotions. Optimizing trade promotions enables data-driven insights to maximize promotion effectiveness, aids in designing targeted promotions that align with consumer behavior, and facilitates accurate ROI calculations by assessing market risks.
```

=== Updates to the Trade Promotion Effectiveness Use Case

> This release includes updates to the https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%203b%20-%20Trade%20promotion%20effectiveness/[Trade Promotion Effectiveness^] use case with full and incremental load, which helps visualize your promotions in one integrated platform with pre-built assets that integrate Business Intelligence Tools (BI Tools), such as CRM Analytics, into a TPM platform like Salesforce Consumer Goods Cloud.

Reviewer: ```suggestion
This release updates the https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%203b%20-%20Trade%20promotion%20effectiveness/[Trade Promotion Effectiveness^] use case with full and incremental load, which helps visualize your promotions in one integrated platform with pre-built assets that integrate Business Intelligence Tools (BI Tools), such as CRM Analytics, into a TPM platform like Salesforce Consumer Goods Cloud.
```
